### PR TITLE
feat: repo arrangement and initial shared prettier config

### PR DESCRIPTION
**Pull request by rrubenich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+```
+{
+    // ...
+    "prettier": "@wealthsystems/prettier-config"
+}
+```
+
+If you don't want to use `package.json` or want to extend this configuratin, a guide is available [here](https://prettier.io/docs/en/configuration.html#sharing-configurations).
+
```
Comment (arslivinski) [inline README.md:31]: remove unnecessary blank lines

File: index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
```
Comment (arslivinski) [inline index.js:1]: Use single quotes

**Review by arslivinski — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rrubenich) [reply README.md:31]: done

File: index.js (hunk 2 above)
Comment (rrubenich) [reply index.js:1]: done :rocket:

**Review by arslivinski — APPROVED** [review]

**Pull request merged**